composer.json

```diff
@@ -28,7 +28,7 @@
     "docs": "https://pimcore.com/docs/latest/"
   },
   "require": {
-    "php": "~8.1.0 || ~8.2.0",
+    "php": "~8.1.0 || ~8.3.2",
```

Comment:
We must not drop hp 8.2 support, 8.3 should be added additionally

Reply:
Also: is there anything wrong with PHP `8.3.0` and `8.3.1`?